app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -118,6 +118,77 @@ async def colormap_symbology(
     return [job], new_asset_uri
 
 
+async def date_conf_intensity_multi_8_symbology(
+    dataset: str,
+    version: str,
+    pixel_meaning: str,
+    source_asset_co: RasterTileSetSourceCreationOptions,
+    zoom_level: int,
+    max_zoom: int,
+    jobs_dict: Dict,
+) -> Tuple[List[Job], str]:
+    """Create Raster Tile Set asset which combines date_conf raster and
+    intensity raster into one.
+
+    At native resolution (max_zoom) it will create intensity raster
+    based on given source. For lower zoom levels it will resample higher
+    zoom level tiles using bilinear resampling method. Once intensity
+    raster tile set is created it will combine it with source
+    (date_conf) raster into RGB-encoded raster.
+    """
+    intensity_co = source_asset_co.copy(deep=True, update={"data_type": DataType.uint8})
+    return await _date_intensity_symbology(
+        dataset,
+        version,
+        pixel_meaning,
+        intensity_co,
+        zoom_level,
+        max_zoom,
+        jobs_dict,
+        # What we want is a value of 55 (max intensity) anywhere there is
+        # an alert in any system. We can't just do
+        # "((A > 0) | (B > 0) | (C > 0)) * 55" because "A | B" includes only
+        # those values unmasked in both A and B. So first replace masked
+        # values with 0 and then re-mask them later
+        "np.ma.array([((A.filled(0) >> 1) > 0) * 55, ((B.filled(0) >> 1) > 0) * 55, ((C.filled(0) >> 1) > 0) * 55])",
```

Comment:
would it be possible to write complex calcs like these as a multiline string with indenting, so it's easier to mentally parse what's going on?

Reply:
I think I've improved it quite a bit.